src/pykeen/models/unimodal/ermlp.py

```diff
@@ -43,24 +37,46 @@ class ERMLP(ERModel):
         embedding_dim=DEFAULT_EMBEDDING_HPO_EMBEDDING_DIM_RANGE,
     )
 
+    @update_docstring_with_resolver_keys(
+        ResolverKey(name="activation", resolver="class_resolver.contrib.torch.activation_resolver")
```

Comment:
also add the entity initializer and relation initializer?

Reply:
They do not fit yet, since we do not expose the `kwargs` part.